src/main/java/no/idporten/userservice/api/admin/AdminApiController.java

```diff
@@ -122,6 +122,25 @@ public ResponseEntity<UserResource> updateUserAttributes(@UUID(message = "Invali
         return ResponseEntity.ok(apiUserService.updateUserAttributes(id, request));
     }
 
+    @Operation(
+            summary = "Update attributes for user",
+            description = "Update user attributes based on pid. Note that if the user does not exist, it will be created.",
+            tags = {"admin-api"},
+            security = @SecurityRequirement(name = "access_token"),
+            parameters = {
+                    @Parameter(in = ParameterIn.PATH, name = "pid", required = true, description = "User id")
+            })
+    @ApiResponses(value = {
+            @ApiResponse(responseCode = "200", description = "User attributes updated"),
+            @ApiResponse(responseCode = "404", description = "User is not found")
```

Comment:
Vil ikke den nye service-metoden opprette bruker, slik at det ikke er mulig å få en 404 user not found her?